contracts/tg4-engagement/src/multitest/suite.rs

```diff
@@ -45,7 +45,7 @@ impl SuiteBuilder {
     pub fn with_member(mut self, addr: &str, weight: u64) -> Self {
         self.members.push(Member {
             addr: addr.to_owned(),
-            weight,
+            points: weight,
```

Comment:
Was going to point this out, but yeah, fine to handle variables in the follow up PR